RELEASE.md

```diff
@@ -1,123 +1,25 @@
-*   Added the ConcatPlaceholder to tfx.dsl.component.experimental.placeholders.
-*   Changed Span information as a property of ExampleGen's output artifact.
-    Deprecated ExampleGen input (external) artifact.
-*   Added ModelRun artifact for Trainer for storing training related files,
-    e.g., Tensorboard logs. Trainer's Model artifact now only contain pure
-    models (check `tfx/utils/path_utils.py` for details).
-*   Added support for `tf.train.SequenceExample` in ExampleGen:
-    *   ImportExampleGen now supports `tf.train.SequenceExample` importing.
-    *   base_example_gen_executor now supports `tf.train.SequenceExample` as
-        output payload format, which can be utilized by custom ExampleGen.
-*   Added Tuner component and its integration with Google Cloud Platform as
-    the execution and hyperparemeter optimization backend.
-*   Switched Transform component to use the new TFXIO code path. Users may
-    potentially notice large performance improvement.
-*   Added support for primitive artifacts to InputValuePlaceholder.
-*   Supported multiple artifacts for Trainer and Tuner's input example Channel.
-*   Supported split configuration for Trainer and Tuner.
-*   Supported split configuration for Evaluator.
-*   Supported split configuration for StatisticsGen, SchemaGen and
-    ExampleValidator. SchemaGen will now use all splits to generate schema
-    instead of just using `train` split. ExampleValidator will now validate all
-    splits against given schema instead of just validating `eval` split.
-*   Component authors now can create a TFXIO instance to get access to the
-    data through `tfx.components.util.tfxio_utils`. As TFX is going to
-    support more data payload formats and data container formats, using
-    `tfxio_utils` is encouraged to avoid dealing directly with each combination.
-    TFXIO is the interface of [Standardized TFX Inputs](
-    https://github.com/tensorflow/community/blob/master/rfcs/20191017-tfx-standardized-inputs.md).
-*   Added experimental BaseStubExecutor and StubComponentLauncher to test TFX
-    pipelines.
-*   Added experimental TFX Pipeline Recorder to record output artifacts of the
-    pipeline.
-*   Supported multiple artifacts in an output Channel to match a certain input
-    Channel's artifact count. This enables Transform component to process
-    multiple artifacts.
-*   Transform component's transformed examples output is now optional (enabled
-    by default). This can be disabled by specifying parameter
-    `materialize=False` when constructing the component.
-*   Supported `Version` spec in input config for file based ExampleGen.
-*   Added custom config to Transform component and made it available to
-    pre-processing fn.
-*   Supported custom extractors in Evaluator.
-*   Deprecated tensorflow dependency from MLMD python client.
-*   Supported `Date` spec in input config for file based ExampleGen.
+*   Added Python 3.8 support.
```

Comment:
I would not highlight this since it's not reliably offered. It works as a happy surprise, rather than guarantee.

So replace with an 'N/A'?